teaser/data/output/modelicatemplate/AixLib/AixLib_ThermalZone

```diff
@@ -0,0 +1,95 @@
+within ${mod_prj}.${bldg.name};
+model ${bldg.name}
+  "This is the simulation model of ${bldg.name}"
+
+% if zone == "ThermalZone":
+  AixLib.ThermalZones.ReducedOrder.ThermalZone.ThermalZone thermalZone(
+    redeclare package Medium = Modelica.Media.Air.SimpleAir,
+    zoneParam=
+    ${mod_prj}.${bldg.name}.${bldg.name}_DataBase.${bldg.name}_${zone.name
+	  .replace(" ", "")}(),
+    ROM(extWallRC(thermCapExt(each der_T(fixed=true))),
+        intWallRC(thermCapInt(each der_T(fixed=true)))),
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    T_start=${zone.t_inside})
+    "Thermal zone"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}})));
+% else:
+  AixLib.ThermalZones.ReducedOrder.ThermalZone.ThermalZoneEquipped thermalZone(
+    redeclare package Medium = Modelica.Media.Air.SimpleAir,
+    zoneParam=
+    ${mod_prj}.${bldg.name}.${bldg.name}_DataBase.${bldg.name}_${zone.name
+	  .replace(" ", "")}(),
+    ROM(extWallRC(thermCapExt(each der_T(fixed=true))),
+        intWallRC(thermCapInt(each der_T(fixed=true)))),
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    T_start=${zone.t_inside})
+    "Thermal zone"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}})));
+
+  AixLib.BoundaryConditions.WeatherData.ReaderTMY3 weaDat(
+    calTSky=AixLib.BoundaryConditions.Types.SkyTemperatureCalculation.HorizontalRadiation,
+    computeWetBulbTemperature=false,
+    filNam=Modelica.Utilities.Files.loadResource(
+      "${weather}"))
+    "Weather data reader"
+    annotation (Placement(transformation(extent={{-92,20},{-72,40}})));
+
+  AixLib.BoundaryConditions.WeatherData.Bus weaBus
+    "Weather data bus"
+    annotation (Placement(transformation(extent={{-78,-20},{-44,12}}),
+    iconTransformation(extent={{-70,-12},{-50,8}})));
+
+  Modelica.Blocks.Sources.CombiTimeTable tableInfiltrationRate(table=[
+  0
+  % for tz in bldg.thermal_zones:
+    ,${tz.infiltration_rate} ${';' if (loop.last) else ''}
+  %endfor
+  3600
+  % for tz in bldg.thermal_zones:
+    ,${tz.infiltration_rate}
+  %endfor
+  ])
+    annotation (Placement(transformation(extent={{-92,-40},{-72,-20}})));
+
+  Modelica.Blocks.Sources.CombiTimeTable tableInternalGains(
+    extrapolation=Modelica.Blocks.Types.Extrapolation.Periodic,
+    table=[0,0,0,0; 3600,0,0,0; 7200,0,0,0; 10800,0,0,0; 14400,0,0,0; 18000,0,0,
```

Comment:
Currently the same problem as for Annex60 models. We create the boundary condition tables in `logic.simulation.aixlib` for each building containing all zones. For single zones, we would need to adapt this to write a file per zone. For this, we would need to give this function the information what model is used. Thus, we would need to shift this function to `aixlib_output`, resp. to a output module that serves for both worlds, Annex60 and AixLib